hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/ServiceFactory.java

```diff
@@ -22,7 +22,7 @@
 import java.util.ServiceLoader;
 
 /**
- * This class provides the ability to load {@link Service} implementations at runtime. The Java SPI
+ * This class provides the ability to load {@link RpcService} implementations at runtime. The Java SPI
```

Comment:
Should the class be renamed to RpcServiceFactory?

Reply:
Yes, that's a good idea.

Reply:
Done